### ISSUE TITLE
Documentation Gaps Tracker

### ISSUE THREAD

**Issue by msaroufim** [pytorch/pytorch#93567]
Keeping Track of documentation that's missing but really needs to exist

- [x] How does dynamo work with distributed https://github.com/pytorch/torchdynamo/pull/1756
- [x] Why dynamo vs alternatives https://github.com/pytorch/torchdynamo/pull/1756
- [x] bug.yml template https://github.com/pytorch/torchdynamo/pull/1741
- [x] FAQ.md https://github.com/pytorch/torchdynamo/pull/1711
- [x] Polish pitch, quick start, custom backends and internals https://github.com/pytorch/torchdynamo/pull/1742
- [x] Benchmarking notes
- [x] Where should people add the decorator: training loop, step, module, module.forward?
- [x] A more interesting getting started guide
- [x] How to disable optimizations
- [x] Training as a superset of inference
- [x] Accuracy debugging
- [x] Dynamic shapes, recompilation UX (powers of 2), partial recompilation
- [x] Make it clearer that biggest benefits will be on A100 with AMP
- [x] Inductor internals
- [ ] Just enough Triton to be dangerous
- [x] Videos?

Lett me know if you think I'm missing anything

cc @ezyang @soumith @wconstab @ngimel @bdhirsh
